clouddriver-kubernetes-v2/src/test/resources/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/manifest/crd-manifest.json

```diff
@@ -8,5 +8,6 @@
     "name": "default-custom1"
   },
   "spec": {
+    "template": "template1"
```

Comment:
It might be better to add a new manifest with the properties you want instead of updating this existing one.  The reason I suggest this is that now we're losing test coverage of the case where `spec.template` is `null`.  It might be good to leave this one having `spec.template` `null` and then add another one with `spec.template` a String and test that neither throws an NPE.

Reply:
thanks for your help and feedback